client/landing/stepper/index.tsx

```diff
@@ -63,49 +56,29 @@ function initializeCalypsoUserStore( reduxStore: any, user: CurrentUser ) {
 	reduxStore.dispatch( requestSites() );
 }
 
-interface configurableFlows {
-	flowName: string;
-	pathToFlow: Flow;
+function determineFlow() {
+	const flowNameFromPathName = window.location.pathname.split( '/' )[ 2 ];
+
+	if ( isAnchorFmFlow() ) {
```

Comment:
What do you think if we check the `isAnchorFmFlow()` before reading the location.pathname?